console/src/main/java/org/eclipse/rdf4j/console/command/Verify.java

```diff
@@ -135,4 +231,18 @@ private String parseDataPath(String... tokens) {
 		}
 		return dataPath.toString();
 	}
+
+	/**
+	 * Write SHACL validation report to a file
+	 * 
+	 * @param model      report
+	 * @param reportFile file name
+	 */
+	private void writeReport(Model model, String reportFile) {
+		try (Writer w = Files.newBufferedWriter(Paths.get(reportFile))) {
+			Rio.write(model, w, RDFFormat.TURTLE);
```

Comment:
Can you enable the pretty print btw? (I'm not sure how to do it from Rio.write myself).

Reply:
There's a Rio.write method that takes a WriterConfig as the 4th argument, so that should be possible